tests/uptane_test.cc

```diff
@@ -755,6 +755,34 @@ TEST(SotaUptaneClientTest, UptaneSecondaryAdd) {
   boost::filesystem::remove_all(uptane_test_dir);
 }
 
+/**
+  * \verify{\tst{149}} Check that basic device info sent by aktualizr on provisioning are on server
+*/
+TEST(SotaUptaneClientTest, provision_on_server) {
+  Config config("tests/config_tests_prov.toml");
+  std::string server = "tst149";
+  Utils::copyDir("tests/test_data", uptane_test_dir);
+  config.uptane.metadata_path = "tests";
+  config.tls.certificates_directory = uptane_test_dir;
+  config.provision.server = server;
+  config.uptane.director_server = server + "/director";
+  config.uptane.repo_server = server + "/repo";
+
+  config.uptane.device_id = "tst149_device_id";
+  config.uptane.primary_ecu_hardware_id = "tst149_hardware_identifier";
+  config.uptane.primary_ecu_serial = "tst149_ecu_serial";
+
+  event::Channel events_channel;
+  command::Channel commands_channel;
+  FSStorage storage(config);
+  HttpFake http(uptane_test_dir);
+  commands_channel << boost::make_shared<command::GetUpdateRequests>();
+  commands_channel << boost::make_shared<command::Shutdown>();
+  Uptane::Repository repo(config, storage, http);
+  SotaUptaneClient up(config, &events_channel, repo);
+  up.runForever(&commands_channel);
+}
```

Comment:
I added it because I got an error that boost cant copy cred.zip file because it is already exists. I think some previous tests did not  clean that directory